openff/interchange/interop/openmm/_topology.py

```diff
@@ -173,6 +186,26 @@ def to_openmm_topology(
                 order=bond_order,
             )
 
+    if has_virtual_sites and not collate:
+        # TODO: This structure is imperfect, no clue how to handle chains here
+        virtual_site_chain = openmm_topology.addChain(atom_chain_id)
+
+        for molecule_index, molecule in enumerate(topology.molecules):
+            virtual_sites_in_this_molecule: list[VirtualSiteKey] = molecule_virtual_site_map[molecule_index]
+
+            # make a new "residue" for each molecule which has virtual sites
+            if len(virtual_sites_in_this_molecule) > 0:
+                this_virtual_site_residue = openmm_topology.addResidue("VS", virtual_site_chain)
```

Comment:
Would it make sense to name these residues by their parent residue name? The use case I'm thinking of here is let's say you want to select all your waters, you'd go "resname WAT HOH SOL" and know you're picking up all the waters and their virtual sites. Similarly if you knew you had a ligand named LIG, then your user experience (especially when you output to a PDB from an openmm topology via something like mdtraj) is a lot cleaner.